migration/migration.go

```diff
@@ -229,6 +229,7 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr *models.GormWork
 	if err := createOrUpdatePlannerItemExtention(models.SystemBug, ctx, witr, db); err != nil {
 		return err
 	}
+	witr.ClearCache() // Clear the WIT cache after updating existing WITs
```

Comment:
It would require to expose the cache outside of the models package. Which could be more error prone (if someone modifies the cache outside of models).